### PR TITLE
issue: 1094990 Rename aux folder to be cloned correctly on Windows

### PR DESCRIPTION
**Pull request by olgk**
In Windows AUX is predefined name in filesystems so should be prohibited
from user applications usage. aux was introduced as place for internal
different VMA subsystems unit and behavior tests. To fix the issue on
cloning from github under Windows it was renamed to "mix".

Signed-off-by: Oleg Kuporosov <olegk@mellanox.com>

### CODE REVIEW

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/accl-libvma-pr/1567/ for details (Mellanox internal link).

File: tests/gtest/Makefile.am [hunk 1]
```diff
@@ -18,13 +18,13 @@ gtest_SOURCES = \
 	common/gtest-all.cc \
 	common/sys.cc \
 	\
-	common/base.cc \
+	/common/base.cc \
```
Comment (ilansmith) [inline tests/gtest/Makefile.am]: `/common/...` ???

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/accl-libvma-pr/1570/ for details (Mellanox internal link).

**Comment by ilansmith** [comment]
merged #440

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/accl-libvma-pr/1585/ for details (Mellanox internal link).

**Pull request closed without merging**